action.sh

```diff
@@ -35,6 +35,11 @@ fi
 if [[ -z "$REQUIRED_ROLES" ]]; then
     echo "WARNING: missing REQUIRED_ROLES, consider adding them to the action config"
 fi
+if [[ -z "$EXCLUDE_TABLES" ]]; then
+    if [ "$TOOL_TYPE" = "rubenv-sql-migrate" ]; then
+        export EXCLUDE_TABLES=migrations
```

Comment:
is it alright if table `migrations` is excluded this way instead of explicitly mentioning in the sql ?

Reply:
I don't have any issues with extracting it here, but the logic of this if statement doesn't make sense. You should be able to specify excluded tables for any tool, so presumably you'd have to append "migrations" when using rubenv-sql-migrate regardless of whether others were provided or not

Reply:
I'm not sure i'm following, if `EXCLUDE_TABLES` is set then this just accept that value regardless of the tool, otherwise it only defaults to `migrations` if the tool is `rubenv-sql-migrate` and this is purely for backward compatibility.